bokeh/models/glyphs.py

```diff
@@ -19,7 +20,19 @@ class Glyph(PlotObject):
 
     """
 
-    visible = Bool(help="""
+    def __init__(self, **kwargs):
+        props = dict()
+        if "line_alpha" in self.__class__.__dict__.keys():
+            props.update(themes.default['line_defaults'])
+        if "fill_alpha" in self.__class__.__dict__.keys():
+            props.update(themes.default['fill_defaults'])
+        if "text_alpha" in self.__class__.__dict__.keys():
+            props.update(themes.default['text_defaults'])
+        props.update(kwargs)
+        super(Glyph, self).__init__(**props)
```

Comment:
I think I'd actually like the mixins to annotate some hidden attribute about what line, fill and text props are present (with their prefixes), so that it can be properly inspected,  but `hasattr` will do for now.